md/howto-debugcustompage.md

```diff
@@ -0,0 +1,89 @@
+# How to Debug a custom page ? 
```

Comment:
Generally: Remove all "spaces" before punctuation: "?", ":" namely.
Use "Bonita BPM Studio", "Bonita BPM Portal", "Bonita BPM Engine" each first time you use it in the page.
Then you can switch to "the studio", "the portal", the engine".
"The UI Designer" is always "the UI Designer, because it's already so long a name.